### ISSUE TITLE
The version of youtube-revanced-extended seems to be wrong

### ISSUE THREAD

**Issue by sorangegu** [j-hc/revanced-magisk-module#475]
The version of youtube-revanced-extended seems to be wrong, it's lower than the last one, which was v19.25.39, this one is v19.16.39

**Comment by acwcanada** [comment]
Not just version is wrong as like you said it goes backward. Even uninstall the previous version 19.25.39 to install this one, Magisk still failed to get it installed, so more problem of the module.

**Comment by j-hc** [comment]
no its not wrong. check supported versions in rvmm-config-gen instead of opening false issues.

> Magisk still failed to get it installed, so more problem of the module.

send the flash logs from magisk then, instead of complaining about a non-existent issue

**Comment by acwcanada** [comment]
> no its not wrong. check supported versions in rvmm-config-gen instead of opening false issues.
> 
> > Magisk still failed to get it installed, so more problem of the module.
> 
> send the flash logs from magisk then, instead of complaining about a non-existent issue

Ok, I am sending two flash logs, the 1st one is updating from v19.25.39, and the 2nd one is after remove the older version and try to flash this one.
[magisk_install_log_2024-07-18T21.16.31.log](https://github.com/user-attachments/files/16302034/magisk_install_log_2024-07-18T21.16.31.log)
[magisk_install_log_2024-07-18T21.22.25.log](https://github.com/user-attachments/files/16302035/magisk_install_log_2024-07-18T21.22.25.log)

**Comment by j-hc** [comment]
you shouldnt uninstall youtube while the module is active. try rebooting

**Comment by acwcanada** [comment]
> you shouldnt uninstall youtube while the module is active. try rebooting

I didn't uninstall youtube while the module is active. Reboot doesn't help. Like I mentioned, I tried first by updating the module on Magisk with v19.25.39 installed, then it failed as shown in the 1st log. Then I removed the v19.25.39 in Magisk, reboot and install the v19.16.39 on Magisk to get the 2nd log. To prove nothing wrong with my system, I went back to install v19.25.39 without any issue, please see the 3rd log of this action.
[magisk_install_log_2024-07-18T22.01.52.log](https://github.com/user-attachments/files/16302305/magisk_install_log_2024-07-18T22.01.52.log)

**Comment by rjaakash** [comment]
> > you shouldnt uninstall youtube while the module is active. try rebooting
> 
> I didn't uninstall youtube while the module is active. Reboot doesn't help. Like I mentioned, I tried first by updating the module on Magisk with v19.25.39 installed, then it failed as shown in the 1st log. Then I removed the v19.25.39 in Magisk, reboot and install the v19.16.39 on Magisk to get the 2nd log. To prove nothing wrong with my system, I went back to install v19.25.39 without any issue, please see the 3rd log of this action. [magisk_install_log_2024-07-18T22.01.52.log](https://github.com/user-attachments/files/16302305/magisk_install_log_2024-07-18T22.01.52.log)

No you are still doing wrong.

1. Remove the module from magisk
2. Reboot device 
3. Uninstall the app from app drawer
4. If you detach yt & ytm by using zygisk-detach then untick that
5. Install official yt & ytm from play store and then uninstall again 
6. Now install the new module update 

Hope now the issue will be solved

Btw, steps 4 & 5 are not necessary but you can do for 100% solution.

**Comment by acwcanada** [comment]
> > > you shouldnt uninstall youtube while the module is active. try rebooting
> > 
> > 
> > I didn't uninstall youtube while the module is active. Reboot doesn't help. Like I mentioned, I tried first by updating the module on Magisk with v19.25.39 installed, then it failed as shown in the 1st log. Then I removed the v19.25.39 in Magisk, reboot and install the v19.16.39 on Magisk to get the 2nd log. To prove nothing wrong with my system, I went back to install v19.25.39 without any issue, please see the 3rd log of this action. [magisk_install_log_2024-07-18T22.01.52.log](https://github.com/user-attachments/files/16302305/magisk_install_log_2024-07-18T22.01.52.log)
> 
> No you are still doing wrong.
> 
> 1. Remove the module from magisk
> 2. Reboot device
> 3. Uninstall the app from app drawer
> 4. If you detach yt & ytm by using zygisk-detach then untick that
> 5. Install official yt & ytm from play store and then uninstall again
> 6. Now install the new module update
> 
> Hope now the issue will be solved
> 
> Btw, steps 4 & 5 are not necessary but you can do for 100% solution.

I know this would work but it is not real issue here (the method is just to downgrade to a older version). Why the version is downgraded? In May 21st, he already released version 19.16.39 , with a few more upgrades: 19.20.34 on May 30, 19.21.40 on June 12 , v19.25.39 is released two weeks ago. And now it is going back to v19.16.39, why? This is entire reason why it can't be updated or installed as OS has detected a downgrade.

**Comment by mingaldrichgan** [comment]
This is likely because of https://github.com/inotia00/ReVanced_Extended/issues/2241.

TL;DR: There were patches that did not work on more recent versions, so support for those versions were dropped for now.

**Comment by acwcanada** [comment]
> This is likely because of [inotia00/ReVanced_Extended#2241](https://github.com/inotia00/ReVanced_Extended/issues/2241).
> 
> TL;DR: There were patches that did not work on more recent versions, so support for those versions were dropped for now.

I read it and understand. However, there are alot of devices such as Google pixel series, Youtube is a system app for those devices, with their July update releases of Android 14 and Android 15, YT version in thier system are all higher than 19.16.39, even you uninstall the updateds, they all would still give downgrade version error when you are updating or fresh install this v19.16.39.

**Comment by Xrdem** [comment]
I have same issue with @acwcanada ,I can confirm error still on going on.I have same log output.Youtube doesnt downgrade bellow 19.17.34 for mine.I tryed with updated or without updated but no luck.btw my phone is redmi 13

**Comment by Koehlix** [comment]
I updated on Android 15 today, now i have the same issue. Is there any fix?

**Comment by FakeStu** [comment]
I just updated to Android 15 and I'm having the same problem, is there any way to solve this?
